app/controllers/subscriptions_controller.rb

```diff
@@ -3,15 +3,16 @@ class SubscriptionsController < ApplicationController
   before_action :assign_attributes
 
   def new
-    @back_url = @frequency.blank? ? govuk_url : url_for(action: :new, topic_id: @topic_id)
     @title = GdsApi.email_alert_api
       .get_subscriber_list(slug: @topic_id)
       .to_h.dig("subscriber_list", "title")
     if @frequency.present?
       return frequency_form_redirect unless valid_frequency
 
+      @back_url = url_for(host: Plek.new.website_root, action: :new, topic_id: @topic_id)
```

Comment:
This shouldn't need to be an absolute url (and would likely cause problems in dev, flipping people onto the www hostname) - I think path_for would work? 

Was there a reason you added host? As I would assume it would work out the correct one based off the request.

Reply:
Oh yes, you're right! I was trying to be consistent with the other uses of "url_for" in the controller, on the assumption they do something special. I've rewritten the commit to remove it instead.